libsolidity/interface/StandardCompiler.cpp

```diff
@@ -125,7 +125,7 @@ Json::Value formatErrorWithException(
 	// TODO: consider enabling color
 	string formattedMessage = SourceReferenceFormatter::formatExceptionInformation(
 		_exception,
-		_type,
```

Comment:
The `_type` variable is used in line `Json::Value formatError` function. I'm not sure if it would be the right thing to constrain the `variable` there to just the 3 types allowed in the enum Severity. 

If however, we are to do it, perhaps it would be better to make a different issue out of it ?